src/prescriptions/prescription-tab-lists.component.tsx

```diff
@@ -154,7 +153,11 @@ const PrescriptionTabLists: React.FC = () => {
                                   <TableExpandRow {...getRowProps({ row })}>
                                     {row.cells.map((cell) => (
                                       <TableCell key={cell.id}>
-                                        {cell.value}
+                                        {cell.id.endsWith("created")
+                                          ? formatDatetime(
+                                              parseDate(cell.value)
+                                            )
+                                          : cell.value}
```

Comment:
I did the formatting here so that the underlying raw ISO string date value is used for sorting (and not the formatted day, which resulted in things being sorted alphabetically)... ideally, I think we should be parsing into a Date object so that that the Cell value would be a Date, not a string containing the raw ISO value) but this seemed to break the sort of the DataTable.